src/main/resources/ath-container/run.sh

```diff
@@ -80,4 +80,4 @@ fi
 
 set -x
 
-BROWSER=$browser JENKINS_WAR=$war $MVN $extra_args "$@"
+BROWSER=$browser JENKINS_LOCAL_HOSTNAME=jenkins.127.0.0.1.sslip.io JENKINS_WAR=$war $MVN $extra_args "$@"
```

Comment:
should this be configurable?

Reply:
we could do it a different way - not modify the files and just start docker with the environment variable and the host I guess.